lua/nvim-tree/actions/dispatch.lua

```diff
@@ -5,43 +5,51 @@ local M = {}
 
 local Actions = {
```

Comment:
My first instinct is was to move each of these to an dispatch.lua in each directory however that adds no value.

Reply:
i guess so :) i still want to improve this module since it's a bit clunky imo.